packages/orchestrator-ui-components/src/components/WfoError/WfoError.tsx

```diff
@@ -27,7 +27,7 @@ export const WfoErrorWithMessage = ({ error }: WfoErrorWithMessageProps) => {
     const t = useTranslations('common');
     const { theme } = useOrchestratorTheme();
     const message =
-        error.length > 0
+        error && error.length > 0
```

Comment:
You can use the `?` to not repeat yourself here

Reply:
TS is not happy

`TS18048: error.length is possibly undefined`